mailpoet/assets/js/src/wizard/steps/pitch_mss_step/first_part.tsx

```diff
@@ -0,0 +1,81 @@
+import { useHistory, useParams } from 'react-router-dom';
+import { external, Icon } from '@wordpress/icons';
+import { Heading } from '../../../common/typography/heading/heading';
+import { MailPoet } from '../../../mailpoet';
+import { List } from '../../../common/typography/list/list';
+import { Button } from '../../../common';
+import { OwnEmailServiceNote } from './own_email_service_note';
+
+type MSSStepFirstPartPropType = {
+  subscribersCount: number;
+  finishWizard: (redirect_url?: string) => void;
+};
+
+function MSSStepFirstPart({
+  subscribersCount,
+  finishWizard,
+}: MSSStepFirstPartPropType): JSX.Element {
+  const history = useHistory();
+  const { step } = useParams<{ step: string }>();
+
+  const mailpoetAccountUrl = MailPoet.MailPoetComUrlFactory.getPurchasePlanUrl(
+    MailPoet.subscribersCount,
+    MailPoet.currentWpUserEmail,
+    'starter',
+    {
+      ref: 'plugin-wizard',
+      utm_source: 'plugin',
+      utm_medium: 'onboarding',
+      utm_campaign: 'purchase',
+    },
+  );
+
+  return (
+    <>
+      <Heading level={1}>
+        {MailPoet.I18n.t('welcomeWizardMSSFirstPartTitle')}
+      </Heading>
+
+      <div className="mailpoet-gap" />
+      <p>{MailPoet.I18n.t('welcomeWizardMSSFirstPartSubtitle')}</p>
+      <div className="mailpoet-gap" />
+
+      <div className="mailpoet-welcome-wizard-mss-list">
+        <List>
+          <li>{MailPoet.I18n.t('welcomeWizardMSSList1')}</li>
+          <li>{MailPoet.I18n.t('welcomeWizardMSSList2')}</li>
+          {subscribersCount < 1000 ? (
+            <li>{MailPoet.I18n.t('welcomeWizardMSSList3Free')}</li>
+          ) : (
+            <li>{MailPoet.I18n.t('welcomeWizardMSSList3Paid')}</li>
+          )}
+        </List>
+      </div>
+
+      <div className="mailpoet-gap" />
+      <div className="mailpoet-gap" />
+
+      <Button
+        isFullWidth
+        href={mailpoetAccountUrl}
+        target="_blank"
+        rel="noopener noreferrer"
+        onClick={(event) => {
+          event.preventDefault();
```

Comment:
I'd define this as a named function to: 
1) Refrain from defining the function in each render
2) Adding a meaningful name to the function would allow other developers to quickly understand what's being done and why

Reply:
Makes sense. fd5a43a5fd3a10c7a8391a6c578ff77665da0b64 is my attempt at implementing it. When you have a moment, it would be great if you could take a look and share your thoughts with me.

Reply:
Thanks for the additional information. I'm assuming you are ok with leaving the code as is, but please let me know if you think I should refactor it to use `useCallback` for some reason.

Reply:
The current code is ok.